### PR TITLE
Update CMakeLists.txt to specify correct library path for installed layer json files

### PR DESCRIPTION
**Pull request by shannon-lunarg**
This PR fixes the library_path entry of the installed layer json files by removing the "./" before installing to system directories.

### CODE REVIEW

File: layers/CMakeLists.txt [hunk 1]
```diff
@@ -222,8 +222,17 @@ if(WIN32)
     endif()
 elseif(UNIX)   # UNIX includes APPLE
     foreach(TARGET_NAME ${TARGET_NAMES})
-        install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET_NAME}.json DESTINATION
-            ${CMAKE_INSTALL_DATAROOTDIR}/vulkan/explicit_layer.d
+        # Add targets for JSON file install on Linux. Need to remove the "./" from the library path before installing to system directories.
+        add_custom_target(${TARGET_NAME}-staging-json ALL
+                          COMMAND mkdir -p ${CMAKE_CURRENT_BINARY_DIR}/staging-json
```
Comment (mikew-lunarg) [inline layers/CMakeLists.txt]: I realize the commands are in an `if(UNIX)` block, so they are probably acceptable in this context.  Will it always be OS-specific?

CMake provides platform-agnostic features of those commands (mkdir, cp), see [Command-line Tool Mode](https://cmake.org/cmake/help/v3.12/manual/cmake.1.html#command-line-tool-mode).  sed is a bit trickier, see [Regular Expressions](https://cmake.org/cmake/help/latest/command/string.html#regular-expressions).

**Comment by shannon-lunarg** [comment]
Hmm.. Looks like I need to dig into why this is working on my Linux machine but not in CI.

**Comment by Tony-LunarG** [comment]
My linux system gets an error similar to the travis build, but for object_tracker instead of unique_objects

**Comment by shannon-lunarg** [comment]
Okay, sorry for the delay. The fix is up now and is passing all checks!

**Pull request merged**